### PR TITLE
feat: Added variables for lootruning (Reverted Files)

### PR DESCRIPTION
**Pull request by Zffu**
**Info Variables Changes**

  -  Added the %slc% and %session_lootrun_chests% variables (The amount of opened chests in a lootrun in the playing session)
  -  Added the %lname% and %lootrun_name% variables (The name of the current lootrun)
  -  Added the %sl% and %session_lootruns% variables (The amount of lootruns done in the playing session)
  -  Added the %lc% and %lootrun_chests% variables (The amount of chests loaded in the current lootrun)
  -  Added the %lp% and %lootrun_points% variables (The amount of points loaded in the current lootrun)
  -  Added the %loc% and %lootrun_opened_chests% variables (The amount of opened chest during the current lootrun)

(I reverted the unecessary files from the last pr)

### CODE REVIEW

File: src/main/java/com/wynntils/modules/map/managers/LootRunManager.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import com.wynntils.modules.map.overlays.objects.MapPathWaypointIcon;
 import com.wynntils.modules.map.rendering.PointRenderer;
 
+import jdk.nashorn.internal.ir.Block;
```
Comment (ryanzhoudev) [inline src/main/java/com/wynntils/modules/map/managers/LootRunManager.java]: Random unused import

File: src/main/java/com/wynntils/modules/map/managers/LootRunManager.java [hunk 2]
```diff
@@ -399,5 +470,4 @@ public JsonElement serialize(Vec3i src, Type typeOfSrc, JsonSerializationContext
             return o;
         }
     }
-
-}
+}
```
Comment (ryanzhoudev) [inline src/main/java/com/wynntils/modules/map/managers/LootRunManager.java]: Missing newline at end of file

File: src/main/java/com/wynntils/modules/utilities/instances/InfoFormatter.java [hunk 3]
```diff
@@ -635,5 +661,4 @@ private void cacheHorseData() {
         cache.put("horsetier", Integer.toString(horse.getTier()));
         cache.put("horselevelmax", Integer.toString(horse.getMaxLevel()));
     }
-}
-
+}
```
Comment (ryanzhoudev) [inline src/main/java/com/wynntils/modules/utilities/instances/InfoFormatter.java]: Missing newline

File: src/main/java/com/wynntils/modules/map/managers/LootRunManager.java [hunk 4]
```diff
@@ -102,7 +106,8 @@ public static Optional<LootRunPath> loadFromFile(String lootRunName) {
             reader.close();
             latestLootrun = path;
             latestLootrunName = lootRunName;
-
+            addLootruntoSession();
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/managers/LootRunManager.java:109]: fix naming `addLootrunToSession`

File: src/main/java/com/wynntils/modules/map/managers/LootRunManager.java [hunk 5]
```diff
@@ -358,6 +364,70 @@ public static boolean unloadLootrun(String name) {
...
+
+    public static boolean isLootrunLoaded() {
+        return isLootrunLoaded;
+    }
+
+    public static String getLatestLootrunName() {
+        return latestLootrunName;
+    }
+
+    public static boolean isCheckALootrunChest(BlockPos pos) {
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/managers/LootRunManager.java:391]: Rename the method so it is more understandable

File: src/main/java/com/wynntils/modules/map/managers/LootRunManager.java [hunk 6]
```diff
@@ -358,6 +364,70 @@ public static boolean unloadLootrun(String name) {
...
+                    s = s + key;
+                }
+            }
+            return s;
+        }
+        return "§c§l✖§r";
+    }
+
+    public static int getLootrunChests() {
+       if(latestLootrun != null) {
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/managers/LootRunManager.java:418]:
```suggestion
       if (latestLootrun != null) {
```

File: src/main/java/com/wynntils/modules/map/managers/LootRunManager.java [hunk 7]
```diff
@@ -358,6 +364,70 @@ public static boolean unloadLootrun(String name) {
...
+           return latestLootrun.getChests().size();
+       }
+       return 0;
+    }
+
+    public static int getLootrunPoints() {
+        if(latestLootrun != null) {
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/managers/LootRunManager.java:425]:
```suggestion
        if (latestLootrun != null) {
```

Comment (Zffu) [reply src/main/java/com/wynntils/modules/map/managers/LootRunManager.java:425]: im gonna fix that soon i finished my break

**Pull request closed without merging**